server/apps/rules/routing_rules.py

```diff
@@ -140,6 +143,10 @@ class RoutingRuleSchemeResource(Resource):
                             },
                             'hour_of_day_to': {
                                 'type': 'string'
+                            },
+                            'time_zone': {
+                                'type': 'string',
+                                'nullable': True
```

Comment:
The PR says the default timezone in UTC. if that's the case then this should be
```
'time_zone': {
    'type': 'string',
    'nullable': False,
    'default': 'UTC'
```

I know that this breaks the existing rules. But that's fine we can clean up that collection.